runtime/src/governance/election.rs

```diff
@@ -620,15 +617,13 @@ impl<T: Trait> Module<T> {
 
         let new_stake = Self::new_stake_reusing_transferable(&mut transferable_stake.seat, stake);
 
-        let balance = <balances::Module<T>>::free_balance(&applicant);
+        ensure!(<balances::Module<T>>::can_reserve(&applicant, new_stake.new), "not enough free balance to reserve");
 
-        ensure!(balance >= new_stake.new, "not enough free balance to cover stake");
+        ensure!(<balances::Module<T>>::reserve(&applicant, new_stake.new).is_ok(), "failed to reserve applicant stake!");
```

Comment:
I saw if a method returns `Result`, instead of this:
`ensure!(<balances::Module<T>>::can_reserve(&applicant, new_stake.new), "not enough free balance to reserve");`
they write next:
`<balances::Module<T>>::reserve(&applicant, new_stake.new)?;`

Reply:
Note `?` sign at the end of the method call.

Reply:
yeah I missed that here when refactoring. should be updated on next round of refactoring :+1: